packages/demo/src/demo-building-blocs/VaporComponent.tsx

```diff
@@ -57,12 +57,17 @@ export const VaporComponent: React.FunctionComponent<VaporComponentProps & React
                     {markdown ? (
                         <Markdown className="markdown-documentation" source={markdown} />
                     ) : (
-                        <div>
-                            There are no guidelines for this component yet, click{' '}
-                            <a href={githubMarkdownLink} target="_blank" rel="noopener noreferrer" className="bold">
+                        <div className="body-l-book">
+                            There are no guidelines for <span className="text bold">{page}</span> yet, click{' '}

```

Comment:
Very nice! I didn't see this was already implemented 👍

Reply:
yeah, it was an easy change to add so I figured why not